templates/vertical-standard/collapsible-filters/script/filterlink.js

```diff
@@ -11,5 +11,22 @@ ANSWERS.addComponent('FilterLink', {
     CollapsibleFilters.Helpers.clearSearch();
     collapsibleFiltersInteractions.focusSearchBar();
   },
+  onCreate: () => {
+    const moveFiltersWrapper = (containerClassName) => {
+      document.getElementsByClassName(containerClassName)[0].appendChild(
+        document.getElementsByClassName('js-answersFiltersWrapper')[0]
+      );
+    }
+    if (collapsibleFiltersInteractions.isCollapsibleFiltersView()) {
+      moveFiltersWrapper('Answers-resultsMidContainer');
+    }
+    collapsibleFiltersInteractions.setListenerOnExpandedFiltersBreakpoint((e) => {
+      if (e.matches) {
+        moveFiltersWrapper('Answers-resultsLeftContainer');
+      } else {
+        moveFiltersWrapper('Answers-resultsMidContainer');
+      }
+    });
```

Comment:
what do you think of moving all of this logic into `collapsibleFitlersInteractions` so that it doesn't have to be duplicated for each template?

Reply:
This logic is quite specific to verticalGrid/Standard with its left and mid div, so I'm not sure if it's generic enough to live in `collapsibleFiltersInteractions` since it's not applicable to the rest of the templates

Reply:
eh, after some thoughts, I decided to move it to `collapsibleFiltersInteractions` anyway